layers/+lang/lua/funcs.el

```diff
@@ -23,8 +29,11 @@
     "sl" 'lua-send-current-line
     "sr" 'lua-send-region
     "'" 'lua-show-process-buffer)
-  (pcase lua-backend
-    (`lsp-emmy (spacemacs//lua-setup-lsp-emmy))))
+  (pcase (spacemacs//lua-backend)
```

Comment:
Multiple issues in the `pcase` form.

1. As mentioned above, the function call to `spacemacs//lua-backend` should be inlined here.
 the problem is, one of the allowed value, `lsp-lua`, is not mentioned in the docstring of `lua-backend`.
2. The branch of `lsp-lua` calls `spacemacs//lua-setup-lsp-lua`, but this doesn't make sense at all because it doesn't select any particular backend. So this entire branch, including the function `spacemacs//lua-setup-lsp-lua` should be removed.
3. Then it becomes clear that the entire `spacemacs//lua-backend` is redundant and does nothing.
4. In all branches, the name of the backend could simply be quoted by `'` instead of backquote.

So this `pcase` form should be changed to the following
```elisp
(pcase lua-backend
   ('lsp-emmy-lua (spacemacs//lua-setup-lsp-emmy-lua))
   ('lsp-lua-language-server (spacemacs//lua-setup-lsp-lua-language-server))
   ('lsp-lua-lsp (spacemacs//lua-setup-lsp-lua-lsp)))
```

Reply:
resolved in new submition.